plugins/sbt/graphql-java-codegen-sbt-plugin/sbt-codegen-exmaple/src/main/java/io/github/dreamylost/api/DroidQueryResolver.java

```diff
@@ -0,0 +1,13 @@
+package io.github.dreamylost.api;
```

Comment:
Why do we need to add generated code to the source root?

Reply:
when use client source code for create a open SDK .We need to encapsulate the client

Reply:
Sorry, didn’t get your point why generated classes should be present until he source root.

Reply:
oh, src_managed maybe better in sbt .At present, there is no way to get initialization src_managed. but I don't know much about it. SO. use source at this.

Reply:
The intention of the plugin is to auto-generate classes and do not do the manual copy of them.

Reply:
@benmccann can you please suggest how to add generated classes to the source root in sbt?
I just don't want to keep generated sources in this repo.

Reply:
The original sbt plugin output them to `target/src_managed_graphql`. See here:

https://github.com/kobylynskyi/graphql-java-codegen/blob/4379b02cb3a2f59c6ca854a7ffc38c22e0b61a43/plugins/sbt/src/main/scala/io/github/kobylynskyi/graphql/codegen/GraphQLCodegenSbtPlugin.scala#L65

Reply:
ok, i will update. but crossTarget do not contains scalaVersion?

Reply:
oh, it probably does. I was just suggesting `target/src_managed_graphql` off memory, but it might be something like `target/scala-2.12/src_managed_graphql`. you'd have to test because I can't remember

Reply:
thanks, this is very nice

Reply:
Oh, i have test it, it is `target/scala-2.12/src_managed_graphql`, and scala compiler can find when import it in other java code.This is the same as the Javacode path generated by  protocol buffer that we use. I think this is the right choice.